DependencyInjection/Compiler/CsrfExtensionPass.php

```diff
@@ -29,6 +29,17 @@ public function process(ContainerBuilder $container)
                 $tokenStorageReference = new Reference('security.context');
             }
             $definition->replaceArgument(0, $tokenStorageReference);
+
+            // Symfony <2.8 BC
+            $typeExtensionTag = $definition->getTag('form.type_extension');
+            if (method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')) {
+                $typeExtensionTag['extended-type'] = 'Symfony\Component\Form\Extension\Core\Type\FormType';
+            } else {
+                $typeExtensionTag['alias'] = 'form';
+            }
+            $definition->setTags(array_merge($definition->getTags(), array(
+                'form.type_extension' => $typeExtensionTag,
+            )));
```

Comment:
you still need to detect the Symfony version in ``FOS\RestBundle\Form\Extension\DisableCSRFExtension::getExtendedType`` though